src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -29,7 +38,11 @@
      * @return collection of user, without duplicates
      */
     public List<User> createUsers(final List<String> emails) {
-        return null;
+        return emails.stream()
+                .distinct()
+                .filter(Objects::nonNull)
```

Comment:
If you have filtering and distinct operations in a stream it's always better to do filtering first to pass less values to the distinct stage. However, in this particular case it doesn't make a big difference, so all fine:)